portal-api/src/v1/services/utilisation-report/email-service.ts

```diff
@@ -17,17 +17,35 @@ const getUkefGefReportingEmailRecipients = () => {
   }
 };
 
+/**
+ * Retrieves the TFM UI URL from the environment variables.
+ * @returns The TFM UI URL.
+ * @throws {InvalidEnvironmentVariableError} If the TFM_UI_URL environment variable is not defined or is empty.
+ */
+const getTfmUiUrl = () => {
+  const tfmUiUrl = process.env.TFM_UI_URL;
+  if (!tfmUiUrl) {
+    throw new InvalidEnvironmentVariableError('TFM_UI_URL environment variable is not defined or is empty');
+  }
+
+  return tfmUiUrl;
+};
+
 /**
  * Sends notification email to UKEF GEF reporting email recipients that a utilisation report has been submitted
  * @param bankName - name of the bank
  * @param reportPeriod - period for which the report covers as a string, eg. June 2023
+ * @param tfmHomepageUrl - URL for the TFM homepage
```

Comment:
I don't think this is a param anymore :)

Reply:
Thanks for catching, forgot to remove this after refactoring